requirements/base.in

```diff
@@ -10,6 +10,5 @@ pyjwkest
 pyjwt[crypto]>=2.1.0      # depends on newer jwt.decode and jwt.encode
 python-dateutil>=2.0
 requests>=2.7.0
-rest-condition>=1.0.3
```

Comment:
Remove rest condition from base requirements.

Reply:
A lot of packages use rest-condition via this package, so this should resolve it for a lot of other packages.